assets/js/pages/SettingsPage/SettingsPage.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { useState, useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { Transition } from '@headlessui/react';
+import _ from 'lodash';
```

Comment:
I see in the rest of the codebase we use this import style

```suggestion
import { isUndefined } from 'lodash';
```